front/lib/api/workspace.ts

```diff
@@ -107,7 +107,7 @@ export async function setInternalWorkspaceSegmentation(
 export async function getMembers(
   auth: Authenticator,
   role?: RoleType
-): Promise<UserType[]> {
+): Promise<(UserType & { workspaces: WorkspaceType[] })[]> {
```

Comment:
nit: should we have a `UserWithWorkspaces` type somewhere ?

Reply:
and/or `UserWithoutWorkspaces`

Reply:
done